### PR TITLE
fix: yarn library license issues

### PR DESCRIPTION
**Pull request by abeizn**
### Summary
fix: yarn library license issues

### Does this close any open issues?
Closes na

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: config-ui/.yarn/releases/yarn-3.4.1.cjs [hunk 1]
```diff
@@ -1,34 +1,6 @@
 #!/usr/bin/env node
 /* eslint-disable */
 //prettier-ignore
-/*!
```
Comment (pjfanning) [inline config-ui/.yarn/releases/yarn-3.4.1.cjs:4]: @abeizn Could you provide background as to why it is ok to remove this license header?

File: LICENSE [hunk 2]
```diff
@@ -234,32 +234,24 @@ SOFTWARE.
 
 --------------------------------------------------------------------------------
 
-The following file is distributed under the BSD 2-Clause License:
+The following file is distributed under the Licenses:
```
Comment (pjfanning) [inline LICENSE]: Even if there is a justification for removing the BSD license header in yarn-3.4.1.cjs, this is unclear. What licenses are you referring to?

File: config-ui/.yarn/releases/yarn-3.4.1.cjs (hunk 1 above)
Comment (abeizn) [reply config-ui/.yarn/releases/yarn-3.4.1.cjs:4]: added

File: LICENSE (hunk 2 above)
Comment (abeizn) [reply LICENSE]: updated

File: LICENSE [hunk 3]
```diff
@@ -234,14 +234,11 @@ SOFTWARE.
 
 --------------------------------------------------------------------------------
 
-The following file is distributed under the BSD 2-Clause License:
+The following file is distributed under different licenses:
```
Comment (pjfanning) [inline LICENSE:237]: you should list them all

File: config-ui/.yarn/releases/yarn-3.4.1.cjs (hunk 1 above)
Comment (pjfanning) [reply config-ui/.yarn/releases/yarn-3.4.1.cjs:4]: can you at least add a link for the LICENSE file but ideally you should list the licenses here?

File: LICENSE (hunk 3 above)
Comment (justinmclean) [reply LICENSE:237]: it looks like they have listed them all, although perhaps not quite in the format I would have chosen.

File: config-ui/.yarn/releases/yarn-3.4.1.cjs (hunk 1 above)
Comment (abeizn) [reply config-ui/.yarn/releases/yarn-3.4.1.cjs:4]: I think so, the bsd license should be listed here just like the apache license is listed in other files

File: LICENSE (hunk 3 above)
Comment (abeizn) [reply LICENSE:237]: Hi justin, thank you very much for your question. Regarding the format you mentioned, I would like to ask you to clarify which format you are referring to.

Comment (abeizn) [reply LICENSE:237]: @justinmclean

**Review by d4x1 — APPROVED** [review]

**Pull request merged**